summary.proto

```diff
@@ -35,6 +35,11 @@ message Summary {
 
   repeated XYZComponent xyz_component = 4;
 
+  message NumberCount {
+    required uint32 number = 1;
```

Comment:
What is the semantic meaning of number? Is it the protobuf entry number?

Reply:
It is the number of things in a system and we have count systems with that number.

Reply:
Some clearer names would be really nice. Maybe s/count/num_systems/ that would be much clearer for me. Maybe s/number/metric/.